src/iocore/aio/test_AIO.cc

```diff
@@ -223,11 +223,11 @@ dump_summary()
 
   auto &m = Metrics::getInstance();
 
-  Metrics::IntType *completed = m.lookup(m.lookup("proxy.process.io_uring.completed"));
-  Metrics::IntType *submitted = m.lookup(m.lookup("proxy.process.io_uring.submitted"));
+  Metrics::Counter::AtomicType *completed = m.lookup(m.lookup("proxy.process.io_uring.completed"));
+  Metrics::Counter::AtomicType *submitted = m.lookup(m.lookup("proxy.process.io_uring.submitted"));
```

Comment:
You could have had two more useless casts here.

Reply:
You implying we should have another lookup function, that avoids both lookups here/ This is an unusual use case honestly, the the IdType lookups are kept mostly because of the old APIs and use of "ids" instead of pointers.

We can probably eliminate some of this in v11 once we clean out more old stuff completely. Or, if we could convince Strostrup to allow overloading to take the return type into the resolution.